README.md

```diff
@@ -192,30 +193,34 @@ There is one encryption transform included in the SDK that can be added to your
 ### Conversion
 There are two conversions included in the SDK that can be added to your pipeline. These transforms return a `string`.
 
- - `NewConversion()` - This function returns a `Conversion` instance that is used to access the following conversion functions 
+ - `NewConversion()` - This function returns a `Conversion` instance that is used to access the following conversion functions: 
     - `TransformToXML`  - This function receives an `events.Model` type, converts it to XML format and returns the XML string to the pipeline. 
     - `TransformToJSON` - This function receives an `events.Model` type and converts it to JSON format and returns the JSON string to the pipeline.
 
  ### Compressions
 There are two compression types included in the SDK that can be added to your pipeline. These transforms return a `[]byte`.
 
- - `NewCompression()` - This function returns a `Compression` instance that is used to access the following compression functions 
+ - `NewCompression()` - This function returns a `Compression` instance that is used to access the following compression functions:
     - `CompressWithGZIP`  - This function receives either a `string`,`[]byte`, or `json.Marshaler` type, GZIP compresses the data, converts result to base64 encoded string, which is returned as a `[]byte` to the pipeline.
     - `CompressWithZLIB` - This function receives either a `string`,`[]byte`, or `json.Marshaler` type, ZLIB compresses the data, converts result to base64 encoded string, which is returned as a `[]byte` to the pipeline.
 
+### CoreData Functions
+These are functions that enable interactions with the CoreData REST API. 
+- `NewCoreData()` - This function returns a `CoreData` instance. This `CoreData` instance is used to access the following function(s).
+  - `MarkAsPushed` - This function provides the MarkAsPushed function from the context as a First-Class Transform that can be called in your pipeline. [See Definition Above](#.MarkAsPushed()). The data passed into this function is passed along since all required information is provided on the context (EventId, CorrelationId,etc.. )
```

Comment:
`The data passed into this function is passed along since all required information is provided on the context` isn't clear. 
Suggest `No data is passed into this function since all required information is provided on the context`

Reply:
Data from your pipeline is passed in, you have no control over that, and it is passed along to the next function.

Reply:
Ok, i was thinking of the config data. How about  this `The data passed into this function from the pipeline is passed along unmodified`